pkgs/by-name/op/ophcrack-cli/package.nix

```diff
@@ -0,0 +1,74 @@
+{ lib
+, stdenvNoCC
+, libsForQt5
+, fetchurl
+, fetchpatch
+, autoreconfHook
+, libtool
+, pkg-config
+, zlib
+, openssl
+, freetype
+, gcc
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "ophcrack";
+  version = "3.8.0";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/ophcrack/files/ophcrack/3.8.0/ophcrack-3.8.0.tar.bz2";
+    hash = "sha256-BIpt9XmDo6WjGsfE7BLfFqpJ5lKilnbZPU75WdUK7uA=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    libtool
+    libsForQt5.wrapQtAppsHook
```

Comment:
is this necessary, given `"--disable-gui"` ?

Reply:
this still not address, can build without qt5

Reply:
you are correct they are both not needed but then expat library is needed to build the package successfully.